srcpkgs/zig/template

```diff
@@ -1,8 +1,8 @@
 # Template file for 'zig'
 pkgname=zig
 version=0.6.0
-revision=1
-archs="x86_64*"
+revision=2
+archs="x86_64* aarch64*"
```

Comment:
Is it still that limited? :/

Also, if the final result for `x86_64` doesn't change, you don't need to revbump, since changes to the template will trigger the build for aarch64 anyway.

Reply:
@ericonr I only tested aarch64, but I guess travis will test most stuff. Do you think I should change `archs` and see what happens?

Also, I'll decrease the revbump along with it.

Reply:
I think it'd make sense. Ideally also test programs built using it.

Reply:
Did as much as I could about this, I'll give more details in further comments.